tests/unit/test_client.py

```diff
@@ -139,6 +139,29 @@ def test_client_has_endpoint_url_on_meta(self):
         self.assertEqual(service_client.meta.endpoint_url,
                          'https://foo.bar')
 
+    def test_api_version_is_passed_to_loader_if_provided(self):
+        creator = self.create_client_creator()
+        self.endpoint.host = 'https://foo.bar'
+        specific_api_version = '2014-03-01'
+        creator.create_client(
+            'myservice', 'us-west-2', credentials=self.credentials,
+            api_version=specific_api_version)
+        self.loader.load_service_model.assert_called_with(
+            'myservice', 'service-2', api_version=specific_api_version)
+
+    def test_create_client_class_creates_class(self):
```

Comment:
What is the purpose of adding this test with respect to the what the PR is fixing?